assets/lang/pl_PL.ini

```diff
@@ -138,7 +138,7 @@ OnScreen = Przyciski ekranowe
 Portrait = Pionowo
 Portrait Reversed = Odwrócone pionowo
 PSP Action Buttons = Klawisze akcji PSP
-Rapid fire interval = Rapid fire interval
+Rapid fire interval = 
```

Comment:
This is missing :)

Reply:
Oh right, didnt noticed my bad, will fix later today